### PR TITLE
template wire ups added

### PR DESCRIPTION
**Pull request by upendrasharma221**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/upendrasharma221/wifiwelcome/pull/3" target="_blank">ReviewNinja

**Comment by upendrasharma221** [comment]
[Wifi Welcome: Template wire ups](https://trello.com/c/v6c2Jspq/92-wifi-welcome-template-wire-ups)

File: wifiwelcome/js/saas.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+var x = 1;
```
Comment (shashirepo) [inline wifiwelcome/js/saas.js]: whats this for ?

File: wifiwelcome/index.html [hunk 2]
```diff
@@ -538,9 +534,50 @@ <h1 id="logo" onclick="openMMenu('#changeLogo', 'logo')">
...
+                    <div class="modal-footer">
+                        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                        <button type="button" class="btn btn-primary" onclick="stepOneForHotel('stepOne')">Next</button>
+                    </div>
+                </div><!-- /.modal-content -->
+            </div><!-- /.modal-dialog -->
+        </div><!-- /.modal -->
+
+        <!-- Modal -->
+        <div class="modal fade" id="myModalHotel" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (shashirepo) [inline wifiwelcome/index.html]: never use "myModal", keep it based on context I.e. - branch_model

File: wifiwelcome/index.html [hunk 3]
```diff
@@ -12,41 +12,16 @@
         <link rel="icon" type="image/x-icon" href="favicon.ico" />
         <title class="mojagedit" mojag-element='Title'>SASS Template</title>
 
+        <link rel="stylesheet" type="text/css" href="css/bootstrap.css">
+
         <!-- jQuery library -->
         <script src="js/jquery.min.js"></script>
-        <!-- <script src="js/jquery.min.js" type="text/javascript"></script> -->
-
```
Comment (shashirepo) [inline wifiwelcome/index.html]: why did you remove minifed js and bootstrap

File: wifiwelcome/js/saas.js (hunk 1 above)
Comment (upendrasharma221) [inline wifiwelcome/js/saas.js]: this is to check when initially click on add more field button, it will take 1 and if click again and again it will be incremented and will check condition if(x<8) (means it will allow to add only 8 fields there at a time.

File: wifiwelcome/index.html (hunk 2 above)
Comment (upendrasharma221) [inline wifiwelcome/index.html]: does these names for modals are ok wifiwelcome_asking_model, wifiwelcome_hotel_model ?

File: wifiwelcome/index.html (hunk 3 above)
Comment (upendrasharma221) [inline wifiwelcome/index.html]: that was not needed for the system

File: wifiwelcome/index.html (hunk 2 above)
Comment (shashirepo) [inline wifiwelcome/index.html]: we need to keep them small, so make it branch_model, as those branches for hotels
